.github/workflows/update-from-cli.yaml

```
@@ -0,0 +1,50 @@
+          path: cli
+
+      - name: Update docs
+        run: |
+          mkdir -p src/pages/reference/
+          rm src/pages/reference/cli.mdx
+          cp -r cli/docs/index.md src/pages/reference/cli.mdx
+          rm -rf cli
+
+      - name: Check for changes
+        id: check-changes
+        run: |
```

actionlint (1.7.7)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)